app/build.gradle

```diff
@@ -47,4 +55,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation "androidx.test:runner:1.1.0-alpha4"
     androidTestImplementation "androidx.test.espresso:espresso-core:3.1.0-alpha4"
+    testImplementation 'org.mockito:mockito-core:2.22.0'
```

Comment:
It would be nice to have some kind of order in the dependencies.
For example you could group by scope, like testImplementation, androidTestImplementation, androidTestUtil